aspnet/mvc/models/validation.rst

```diff
@@ -41,172 +41,155 @@ Simply reading through the model reveals the rules about data for this app, maki
 
- .. literalinclude:: validation/sample/MovieIValidatable.cs
+.. literalinclude:: validation/sample/MovieIValidatable.cs
    :language: c#
-   :lines: 35-48
+   :lines: 33-41
    :dedent: 8
-  
+
 Client side validation
 ----------------------
 
-Client side validation is a great convenience for users. It saves time they would otherwise spend waiting for a round trip to the server. In business terms, even a few fractions of seconds multiplied hundreds of times each day adds up to be a lot of time, expense, and frustration. Straightforward and immediate validation enables users to work more efficiently and produce better quality input and output. 
-
-You must have a view with the proper JavaScript script references in place for client side validation to work as you see here. 
-
-.. code-block:: html
+Client side validation is a great convenience for users. It saves time they would otherwise spend waiting for a round trip to the server. In business terms, even a few fractions of seconds multiplied hundreds of times each day adds up to be a lot of time, expense, and frustration. Straightforward and immediate validation enables users to work more efficiently and produce better quality input and output.
 
-  <script src="~/lib/jquery/jquery.js"></script>
-  <script src="~/lib/jquery-validation/dist/jquery.validate.js"></script>
-  <script src="~/lib/jquery-validation-unobtrusive/jquery.validate.unobtrusive.js"></script>
+You must have a view with the proper JavaScript script references in place for client side validation to work as you see here.
 
-MVC uses validation attributes in addition to type metadata from model properties to validate data and display any error messages using JavaScript. When you use MVC to render form elements from a model using `Tag Helpers <https://docs.asp.net/en/latest/mvc/views/tag-helpers/index.html>`_ or `HTML helpers <https://docs.asp.net/en/latest/mvc/views/html-helpers.html>`_ it will add HTML 5 `data- attributes <http://w3c.github.io/html/dom.html#embedding-custom-non-visible-data-with-the-data-attributes>`_ in the form elements that need validation, as shown below. MVC generates the ``data-`` attributes for both built-in and custom attributes. The ``data-val-required`` attribute below contains an error message to display if the user doesn't fill in the release date field, and that message displays in the accompanying ``<span>`` element. You can display validation errors on the client using the relevant tag helpers as shown here:
+.. literalinclude:: validation/sample/Views/Shared/_Layout.cshtml
+   :language: html
+   :lines: 37
+   :dedent: 4
+.. literalinclude:: validation/sample/views/Shared/_ValidationScriptsPartial.cshtml
+   :language: html
```

Comment:
Because this is three `<script>` tags. All are required for client-side validation but one is always useful and is therefore in the layout file. Guess two boxes is :ok: if that's all we can do.